torch/nn/modules/module.py

```diff
@@ -329,7 +329,7 @@ def _tracing_name(self, tracing_state):
         return None
 
     def _slow_forward(self, *input, **kwargs):
-        input_vars = tuple(torch.autograd.function._iter_variables(input))
+        input_vars = tuple(torch.autograd.function._iter_variables_permissive(input))
```

Comment:
Good to have a comment explaining why this one should be permissive.

Reply:
me too. I will try to rediscover it

Reply:
apparently it doesn't need to be permissive. I'll change it back